packages/tour-kit/package.json

```diff
@@ -24,20 +24,28 @@
 	"types": "dist/types",
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
-		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
 		"prepack": "yarn run clean && yarn run build",
 		"watch": "tsc --build ./tsconfig.json --watch",
 		"storybook": "start-storybook"
 	},
 	"dependencies": {
+		"@automattic/components": "workspace:^",
+		"@automattic/viewport": "workspace:^",
 		"@automattic/viewport-react": "workspace:^",
 		"@popperjs/core": "^2.10.2",
+		"@wordpress/base-styles": "^4.1.0",
```

Comment:
I think we are using v4.0.4 in the rest of Calypso, so I would just change these version strings to match other package.json files in the repo, and then run `yarn install` again.